src/main/java/org/cryptomator/ui/controls/FontAwesome5Spinner.java

```diff
@@ -0,0 +1,27 @@
+package org.cryptomator.ui.controls;
+
+import org.cryptomator.ui.common.Animations;
+import org.cryptomator.ui.common.AutoAnimator;
+
+import javafx.beans.NamedArg;
+import javafx.beans.value.ObservableValue;
+
+/**
+ * An animated progress spinner using the {@link FontAwesome5IconView} with the spinner glyph.
+ *
+ * By default, the animation is always played if the icon is visible. To animate on other conditions, use the "spinning" property.
+ */
+public class FontAwesome5Spinner extends FontAwesome5IconView{
+
+	private final AutoAnimator animator;
+
+	public FontAwesome5Spinner(@NamedArg("spinning") ObservableValue<Boolean> spinning) {
```

Comment:
Don't we need a no-args constructor (that calls `this(new SimpleBooleanProperty(true))`)? Didn't try it, but I doubt that the fxml files that use this class can construct it as is.

Reply:
I tried and it works.

Reply:
And now I know why: you added a null check a few lines later. I'd avoid this and rather use explicit constructors without hidden behaviour.

Reply:
Fixed in 2dac9f565fd92421eba2cb173d45264487930a25